pixels-turbo/pixels-invoker-vhive/src/main/java/io/pixelsdb/pixels/invoker/vhive/PartitionedJoinStreamInvoker.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 PixelsDB.
+ *
+ * This file is part of Pixels.
+ *
+ * Pixels is free software: you can redistribute it and/or modify
+ * it under the terms of the Affero GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * Pixels is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * Affero GNU General Public License for more details.
+ *
+ * You should have received a copy of the Affero GNU General Public
+ * License along with Pixels.  If not, see
+ * <https://www.gnu.org/licenses/>.
+ */
+package io.pixelsdb.pixels.invoker.vhive;
+
+import com.alibaba.fastjson.JSON;
+import com.google.common.util.concurrent.ListenableFuture;
+import io.pixelsdb.pixels.common.turbo.Input;
+import io.pixelsdb.pixels.common.turbo.Output;
+import io.pixelsdb.pixels.planner.plan.physical.input.PartitionedJoinInput;
+import io.pixelsdb.pixels.planner.plan.physical.output.JoinOutput;
+import io.pixelsdb.pixels.turbo.TurboProto;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.concurrent.CompletableFuture;
+
+public class PartitionedJoinStreamInvoker extends VhiveInvoker
+{
+    private final Logger log = LogManager.getLogger(PartitionedJoinStreamInvoker.class);
+
+    protected PartitionedJoinStreamInvoker(String functionName)
+    {
+        super(functionName);
+    }
+
+    @Override
+    public Output parseOutput(String outputJson)
+    {
+        return JSON.parseObject(outputJson, JoinOutput.class);
+    }
+
+    @Override
+    public CompletableFuture<Output> invoke(Input input)
+    {
+//        log.info(String.format("invoke PartitionedJoinInput: %s", JSON.toJSONString(input, SerializerFeature.PrettyFormat, SerializerFeature.DisableCircularReferenceDetect)));
```

Comment:
Is this log needed?

Reply:
No for now. Just strictly following the example of ScanInvoker, where there's also a commented out log.info()